app/controllers/wiki_controller.rb

```diff
@@ -266,9 +266,10 @@ def revisions
     @node = Node.find_wiki(params[:id])
     if @node
       @revisions = @node.revisions
-      @revisions = @revisions.where(status: 1) unless current_user && current_user.can_moderate?
+      @revisions = @revisions.where(status: 1).page(params[:page]).per_page(20) unless current_user && current_user.can_moderate?
```

Comment:
oh aha - so this `unless` condition turns off the pagination, right? So then we have to be sure not to treat the collection as paginated. You could add the same condition to line 272?

Reply:
Ohh! 😅  Ya you are right!